src/main/java/com/kodilla/ecommercee/domain/Order.java

```diff
@@ -16,10 +16,7 @@
     @JoinColumn(name = "USER_ID")
     private User user;
 
-    @OneToOne(cascade = CascadeType.ALL,
-            fetch = FetchType.EAGER,
-            targetEntity = Cart.class,
-            mappedBy = "order")
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
W bazie danych relacja powinna być jednokierunkowa raczej. Tu powinno być `mappedBy` (albo z drugiej strony).

Reply:
@interceptor2048 To po drugiej stronie ( w encjiCart) jest mappedBy = "cart". Z encji Order usuwałam mappedBy.